packages/project-access/src/project/mta.ts

```diff
@@ -0,0 +1,31 @@
+import type { Editor } from 'mem-fs-editor';
+import { findFileUp } from '../file/file-search';
+import { FileName } from '../constants';
+import { dirname } from 'path';
+import type { MtaPath } from '../types/mta';
+
+/**
+ * Searches `projectPath` and parent folders.
+ * If mta.yaml file is inside projectPath, this is a special type of MTA project
+ * that created in Fiori generator (Standalone App Router). E.g. Creating a new project that doesn't have
+ * a parent root folder for MTA project, and this project itself is configured
+ * to have deploy target CF and user answered yes to "add to Managed App Router" question.
+ *
+ * @param projectPath Fiori app root folder
+ * @param fs
+ * @returns MtaPath
+ */
+export async function getMtaPath(projectPath: string, fs?: Editor): Promise<MtaPath | undefined> {
+    const mtaPath = await findFileUp(FileName.MtaYaml, projectPath, fs);
+    if (!mtaPath) {
+        return undefined;
+    } else {
+        console.log('before', mtaPath);
```

Comment:
Is this `console.log` a leftover or should this remain in the code
```suggestion
```

Reply:
> Is this `console.log` a leftover or should this remain in the code

Its a left over from debugging the unit test! I will remove it.